egs/swbd/s5c/local/chain/run_tdnn_6z.sh

```diff
@@ -55,7 +55,7 @@ final_layer_normalize_target=0.5
 num_jobs_initial=3
 num_jobs_final=16
 minibatch_size=128
-relu_dim=576
+relu_dim=1024
```

Comment:
Ah, okay, I think I misinterpreted you.

Currently, I propose that 6z be changed to have:
```
--splice-indexes "-2,-1,0,1,2 -1,2 -3,3 -7,2 0" \ 
```
which mostly has splices of size 2 while it before had
```
--splice-indexes "-1,0,1 -1,0,1,2 -3,0,3 -3,0,3 -3,0,3 -6,-3,0 0" \
```
which mostly has splices of size 3.

So the purpose of changing relu_dim from 576 to 768 is so that the input dimension of each affine transform is (4/3)*2 = 8/3 (2 is for the 2 splices as input, 4/3 is 768/576) in the new setup, which is almost 3, for the 3 splices of input for the 6z setup.

But this analysis assumes that the new model has the same number of layers as the cross entropy model. The cross-entropy model has two fewer layers. I hope you agree with me that this is a problem.

I'm still a bit confused, so perhaps the best thing I can say is: look at my splice-indices again. Is this what you wanted? Please make sure. It has two fewer layers than the current chain models. I reduced the number of layers to make the number of layers the same as the cross entropy model.